its/src/org/sonarlint/eclipse/its/StandaloneAnalysisTest.java

```diff
@@ -128,6 +129,19 @@ public void analyze_automatic_workspace_build_disabled() {
     preferences.ok();
   }
 
+  /** See SLE-854: JDT tries to find files with 'Java-like' extensions even if they're not Java */
+  @Test
+  public void test_jdt_java_like_extension_COBOL() {
+    new JavaPerspective().open();
+    var rootProject = importExistingProjectIntoWorkspace("connected", "connected");
+
+    var cobolFile = rootProject.getResource("Test.cbl");
+    openFileAndWaitForAnalysisCompletion(cobolFile);
+
+    assertThat(new SonarLintConsole().getConsoleView().getConsoleText())
+      .doesNotContain("File 'Test.cbl' excluded by 'JavaProjectConfiguratorExtension'");
```

Comment:
Is there a way to make sure that the file was actually analyzed, at least by the secrets analyzer?

Reply:
Done, the test is changed to test for both.